### ISSUE TITLE
Update README.md to include "artName" property for include.js

### ISSUE THREAD

**Issue by donmarvex** [zero-to-mastery/Animation-Nation#1528]
From recent pull requests, contributors have omitted this property because it isn't explicitly stated in the README.md file.

The current documentation has this below. 
```JavaScript
let cards = [
  //  Add your card in this section
  {
    pageLink: "./Art/Joy/triangle/triangle.html",
    imageLink: "./Art/Joy/triangle/triangle.gif",
    author: "Joy",
    githubLink: "https://github.com/royranger",
  },
]
```

However, in the _**include.js**_ file, five properties are used and artName is included among these properties;

```JavaScript
`<img class="art-image" src='${c.imageLink}' alt='${c.artName}' />` +
```

In order to prevent future contributors from omitting the property, it should be updated in the documentation.
```JavaScript
let cards = [
  //  Add your card in this section
  {
    artName: "My art name here",
    pageLink: "./Art/Joy/triangle/triangle.html",
    imageLink: "./Art/Joy/triangle/triangle.gif",
    author: "Joy",
    githubLink: "https://github.com/royranger",
  },
]
```
I can raise a PR to update the docs or we could have another contributor update the docs as a good first issue practice. Please review @sophiabrandt

**Comment by sophiabrandt** [comment]
Please feel free to make a pull request @donmarvex. Good catch on the issue.

**Comment by donmarvex** [comment]
> Please feel free to make a pull request @donmarvex. Good catch on the issue.

Hi @sophiabrandt I have raised the PR to make these changes.

**Comment by sophiabrandt** [comment]
Closed with #1534 , thank you @donmarvex .
